content/common/cli-reference.md

```diff
@@ -1208,3 +1209,28 @@ TRIGGER OPTIONS:
 {{ blockEnd }}
 
 {{ blockEnd }}
+
+### CLI Stability Table
+
+CLI commands have three phases that indicate levels of stability:
+
+- `experimental`: These commands are experiments and may or may not be available in later versions of the CLI.
+- `stabilizing`: These commands have moved out of the `experimental` phase and we are now in the active process of stabilizing them. This includes updating flags, command output, errors, and more.
+- `stable`: These commands have moved out of the `stablizing` phase and will not change until the next major version release.
+
+| Command                         | Stability    |
+| ------------------------------- | ------------ |
+| `spin add`                      | Stable       |
+| `spin build`                    | Stable       |
+| `spin new`                      | Stable       |
+| `spin plugins`                  | Stable       |
+| `spin templates`                | Stable       |
+| `spin up`                       | Stable       |
```

Comment:
What about the `--bindle` flag to `spin up` - do we intend to remove that before 1.0, retain it for the duration of 1.x, or exclude it from the 'stable' state?  It is currently hidden and prints a deprecation warning.

Related: using parcels as component module sources in the manifest.

Reply:
Good point. I think we want to ensure `up` is stable. Continuing this conversation in slack.